src/pages/en/blog/2022-10-02-october-month-of-the-toberfests.md

```diff
@@ -0,0 +1,27 @@
+
+## But what **are** 'toberfests?
+Aside from the German beer festival Oktoberfest, the original and most famous 'toberfest is [Hacktoberfest](https://hacktoberfest.com). Began by Digital Ocean in 2013, Hacktoberfest is, at it's core, a challenge: during the month of October, you must contribute to four participating open-source projects, and the first 40,000 to complete the challenge get a choice of either a t-shirt or a tree planted in your name. 
+
+Inspired by Hacktoberfest, several other groups have held smaller challenges, usually focused on a specific type of project. One of these, which this year Quilt is proud to be sponsoring, is [Modtoberfest](https://modtoberfest.com/), which focuses on contributions to the Minecraft community. While the prizes might not get you the street cred of a Hacktoberfest t-shirt, a tree will be planted for every accepted pull request, and if you submit four accepted pull requests, you will receive a sticker pack containing, among other things, something that no-one else will have: the world's **first** tangible Pineapple. Behold it in all its fruity glory!
+
+![A Pineapple sticker in a dark background](/assets/img/writing/blog/2022-10-02-october-month-of-the-toberfests/toberfest-blog-1.jpeg)
+
+You also get a Quilt logo sticker, a Modfest sticker, a BlanketCon 2022 and some others, to complete the collection.
+
+![Four piles of stickers arranged in a 2x2 grid. From left to right: Quilt logo, Pineapple, Modfest logo, BlanketCon 2022 logo](/assets/img/writing/blog/2022-10-02-october-month-of-the-toberfests/toberfest-blog-2.jpeg)
+
+## How do I start?
+You can start by going to [https://hacktoberfest.com](https://hacktoberfest.com) and/or [https://modtoberfest.com/](https://modtoberfest.com/) and signing up. From there, all you need to do is find a participating project, see what you can contribute, and start coding!
+
+We have a full page with [guidelines for making contributions to Quilt specifically](https://quiltmc.org/toberfests), but a word of advice: maintainers of the projects you contribute to have to officially accept the pull requests that you open in order for it to contribute to your total, and most of them will only accept ones that are high-quality and show effort put in for its own sake, not tiny changes that are clearly made in pursuit of a t-shirt (or the opportunity to be able to hold a Pineapple in one's hands): your contributions should be made with love and genuinely benefit the project that you're submitting to. Make sure to discuss any changes you want to make with the project's maintainers before contributing, and make sure to follow the project's contribution guidelines from the start to minimize their workload in the review phase, and maximize your chances of getting accepted.
```

Comment:
The link should be localized—currently I think the convention is to write direct links without relying on the default English redirect, so
```diff
- [guidelines for making contributions to Quilt specifically](https://quiltmc.org/toberfests)
+ [guidelines for making contributions to Quilt specifically](https://quiltmc.org/en/toberfests) 
```